### DISCUSSION TITLE
Update session on refresh the page

### DISCUSSION THREAD

**Discussion by rahmat-dev** [sergiodxa/remix-auth#250] [Q&A]
I try to update the user manually from database client. When I refresh the page, the user is not updated. How to update user session  when I refresh the page?

**Comment by sergiodxa** [comment]
You control the session storage object so you can call the getSession and commitSession methods to update the user data.

If the session data is stored in a cookie I recommend you to store the minimal info, eg the user ID, and query the rest from the DB.

If you store the session data somewhere else you can edit it manually.
